Sources/NIO/BaseSocketChannel.swift

```diff
@@ -373,6 +561,11 @@ class BaseSocketChannel<T: BaseSocket>: SelectableChannel, ChannelCore {
             return
         }
 
+        guard self.isActive else {
```

Comment:
This should probably check the `lifecycleManager` directly, as it's definitionally called from on the event loop.

Reply:
thanks, yes, forgot to record that change